tests/uefistored/test_cert_inheritance.py

```diff
@@ -325,15 +325,18 @@ def test_set_pool_certificates_partial(self, host):
                 check_disk_cert_md5sum(h, key, pool_auths[key].auth)
             assert not h.file_exists(f'{CERT_DIR}/{missing_key}.auth')
 
-    def test_clear_certificates_from_pool(self, host):
-        keys = ['PK', 'KEK', 'db', 'dbx']
-        pool_auths = generate_keys(as_dict=True)
-        host.pool.install_custom_uefi_certs([pool_auths[key] for key in keys])
-        host.pool.clear_uefi_certs()
+    @pytest.mark.small_vm
+    def test_clear_certificates_from_pool(self, host, uefi_vm_and_snapshot):
+        vm, snapshot = uefi_vm_and_snapshot
+        revert_vm_state(vm, snapshot)
```

Comment:
Should not be necessary. Other tests already revert the VM state in their teardown. That's the contract.

Reply:
not in this class, as it's the only test using a vm in this class.

Reply:
What I mean is:
* either you're executing just this class and the VM is already clean
* or you executed other tests before but they left the VM clean

Conclusion: the VM is already clean of certificates.